### PR TITLE
Feature/ant token styled

### PR DESCRIPTION
**Pull request by GolfGrab**
Migrate to use antd token in styled component
Improve DX with component name in class

### CODE REVIEW

**Comment by GolfGrab** [comment]
mark for review in the future

File: src/modules/order/components/OrderCard.tsx [hunk 1]
```diff
@@ -73,32 +73,25 @@ const FlexBetweenCol = styled.div`
...
-      return "blue";
-    case "ready to serve":
-      return "geekblue";
-    case "success":
-      return "green";
-    case "cancel":
-      return "red";
-  }
-}
+const mapStatusToColor = {
```
Comment (JiMeow) [inline src/modules/order/components/OrderCard.tsx:89]: Make it consistent decide to have " or not

File: src/modules/layout/components/UserTopNav.tsx [hunk 2]
```diff
@@ -1,12 +1,11 @@
-import { transientOptions } from "@/modules/common/transientOptions";
 import styled from "@emotion/styled";
 import { User } from "@phosphor-icons/react";
 import { Button, theme } from "antd";
 import Image from "next/image";
 
 const UserTopNav = () => {
   const {
-    token: { colorPrimary, colorPrimaryBg },
+    token: { colorPrimary },
```
Comment (JiMeow) [inline src/modules/layout/components/UserTopNav.tsx:8]: Does we need this line can we use pattern like `${(props) => props.theme.antd.colorPrimary}` (asking)?

File: src/modules/layout/components/UserBottomNav.tsx [hunk 3]
```diff
@@ -11,14 +11,16 @@ type UserBottomNavProps = {
 const UserBottomNav: React.FC<UserBottomNavProps> = ({ currentPageId }) => {
   const { home, basket, orders } = pages;
 
+  const { token } = theme.useToken();
```
Comment (JiMeow) [inline src/modules/layout/components/UserBottomNav.tsx:14]: What pattern to use? Should pattern be like
`const { token: { colorBgBase },} = theme.useToken();`
then we use colorBgBase not token.colorBgBase

**Review by JiMeow — COMMENTED** [review]
Check my comment please. 😁

**Pull request merged**